compute_tools/src/http/api.rs

```diff
@@ -383,10 +387,12 @@ async fn handle_configure_request(
         // ```
         {
             let mut state = compute.state.lock().unwrap();
-            if state.status != ComputeStatus::Empty && state.status != ComputeStatus::Running {
+            if !matches!(state.status, ComputeStatus::Empty | ComputeStatus::Running) {
```

Comment:
NIT: maybe that's just me, but `!matches!` is harder to read/parse: i) it feels like double-negative (I know that it's a macro); ii) separation of condition and negation. Puts more mental load on me :)

This is probably the reason why you mistakenly changed the log message (or I misunderstood)

Reply:
+1 for plain if from me